src/modules/events/EventsCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Link from "next/link";
+import EventHeader from './EventHeader';
+import Image from "next/image"
+
+type EventCardProps = {
+    id: string;
+    date: string;
+    tags: string[];
+    title: string;
+    image: string;
+    description: string;
+    organizerName: string;
+    attendeesCount: number;
+};
+
+const EventCard = ({
+    id,
+    date,
+    title,
+    image,
+    organizerName,
+    attendeesCount,
+}: EventCardProps) => (
+    <Link href={'#'}>
+        <div className="relative w-full sm:w-64 h-72 rounded-lg overflow-hidden shadow-sm bg-white">
+            {image && (
```

Comment:
If you think image is optional, you can change the type above and also make it optional. 

`image?:string;`